native/src/feature/onboarding/screens/CheckCode.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { View, Text, Button } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+
+export const CheckCode: React.FC<{}> = () => {
+  const navigation = useNavigation();
+
+  return (
+    <>
+      <View>
+        <Text>CheckCode</Text>
+        <Button title={`Next`} onPress={() => navigation.navigate("Root")} />
+      </View>
+    </>
+  );
+};
+
+export default CheckCode;
```

Comment:
Buena iniciativa de crear esta `screen`. 
Sigue esta documentación para utilizar las `Props` correctamente:

https://reactnavigation.org/docs/typescript/#type-checking-screens

```suggestion
export const VerifyPhoneNumberCode: React.FC<Props> = ({ navigation }) => (
      <View>
           <Text>CheckCode</Text>
           <Button title={`Next`} onPress={() => navigation.navigate("Root")} />
      </View>
);

export default CheckCode;
```

Reply:
- He atendido tu comentario, gracias.

Reply:
Hiciste push de tus cambios?
No puedo ver tus nuevos commits